### ISSUE TITLE
Create new contact silently fails when there is no address book

### ISSUE THREAD

**Issue by sedrubal** [owncloud/contacts#563]
### Steps to reproduce
1. install owncloud, create database
2. install contacts app by cloning the git repo and run make or by installing it from the app store
3. go to contacts and try to create a new contact

### Expected behaviour
A default address book should be generated or a hint/warning should appear, when there is no address book

### Actual behaviour
No new contact will be created and no warning appears.

### Server configuration
**Operating system**: `Ubuntu 16.04.1`
**Web server:** `apache 2`
**Database:** `mariadb`
**PHP version:** `php7`
**ownCloud version:** `9.1.3`
**Contacts version:** `1.5.1`
**Updated from an older ownCloud or fresh install:** `fresh install`
**Signing status (ownCloud 9.0 and above):** *I moved `.htaccess` content to apache config, so this is failing for `.htaccess`*
**List of activated apps:**

```
  - activity: 2.3.2
  - comments: 0.3.0
  - contacts: 1.5.1
  - dav: 0.2.7
  - federatedfilesharing: 0.3.0
  - federation: 0.1.0
  - files: 1.5.1
  - files_pdfviewer: 0.8.1
  - files_sharing: 0.10.0
  - files_texteditor: 2.1
  - files_trashbin: 0.9.0
  - files_versions: 1.3.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - gallery: 15.0.0
  - notifications: 0.3.0
  - provisioning_api: 0.5.0
  - systemtags: 0.3.0
  - templateeditor: 0.1
  - updatenotification: 0.2.1
```

**The content of config/config.php:**
```
{
    "system": {
        "instanceid": "ocmixloacgrm",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "172.17.0.2"
        ],
        "datadirectory": "\/var\/www\/data",
        "overwrite.cli.url": "http:\/\/172.17.0.2",
        "dbtype": "mysql",
        "version": "9.1.3.1",
        "dbname": "oc",
        "dbhost": "localhost",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "UTC",
        "installed": true,
        "appstore.experimental.enabled": true
    }
}
```
**Are you using external storage, if yes which one:** `no`
**Are you using encryption:** no
**Are you using an external user-backend, if yes which one:** no

### Client configuration
**Browser:** `Firefox 50.1.0`
**Operating system:** `Fedora 25`

**Comment by m3ntalsp00n** [comment]
This seems fixed to me?

**Comment by sedrubal** [comment]
Yes, now clicking the button has no effect anymore when there is no address book 👍 

But the UX is still quite bad IMHO:

- You open contacts the first time:
  - you see an empty app with a button <kbd>➕ New contact</kbd> which is not working
  - there is an endless loading spinner
  - You have no Idea how to continue
- You find the <kbd>⚙️ Settings</kbd> button and click on it and see an textfield with "address book name ➡️"
- You guess you have to add an address book and type in a name
- You close the settings menu and see exactly the same view as before, but now the <kbd>➕ New contact</kbd> button works as expected...

**Comment by PVince81** [comment]
@Julian1998 any update ?
